shell/run_shell_cmd.go

```diff
@@ -22,5 +23,26 @@ func RunShellCommand(terragruntOptions *options.TerragruntOptions, command strin
 
 	cmd.Dir = terragruntOptions.WorkingDir
 
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, forwardSignals...)
```

Comment:
It's worth mentioning that there is already code listening on some signals in Terragrunt here: https://github.com/gruntwork-io/terragrunt/blob/master/locks/lock.go#L49

Is it OK to listen in more than one place? Should the two be combined?